.travis.yml

```diff
@@ -9,7 +9,7 @@ python:
      - "2.7"
 script:
      - node test.js
-     - pytest
+     - py.test
```

Comment:
Would be better to make verbose, so we see all tests. You can refer to my update beyond #29